modules/setting/setting.go

```diff
@@ -720,6 +723,11 @@ func NewContext() {
 		}
 		UnixSocketPermission = uint32(UnixSocketPermissionParsed)
 	}
+	EnableLetsEncrypt := sec.Key("ENABLE_LETSENCRYPT").MustBool(false)
+	if EnableLetsEncrypt {
```

Comment:
Is this if-statement necessary? If it's false the values are ignored anyhow :thinking: